src/it/java/org/eclipse/microprofile/starter/utils/MPSpec.java

```diff
@@ -50,7 +50,7 @@
     }),
     OPEN_API(new String[][]{
             // Tomee shows only /resilience:, all others show "/data/resilience".
-            new String[]{"openapi", "/resilience"}
+            new String[]{"openapi", "/hello"}
```

Comment:
Why did you change this url?

Reply:
@Emily-Jiang Because /resilience is not present if just OpenAPI is selected, although /hello default endpoint is always present.